quobyte_driver.go

```diff
@@ -38,11 +38,25 @@ func newQuobyteDriver(apiURL string, username string, password string, quobyteMo
 	return driver
 }
 
+func (driver quobyteDriver) StripVolumeName(requestName string) (strippedVolumeName string, subdirPath string) {
```

Comment:
The method can be private (starting with a lower case) ?

Reply:
Done